src/avfoundation.cs

```diff
@@ -1077,15 +1353,22 @@ interface AVAudioMixerNode : AVAudioMixing {
 	interface AVAudioOutputNode {
 
 	}	
+	
+	// AudioBufferList is binded as AudioBuffers which is present in AudioToolbox which is not present on the watch. bug filled: 
+	// https://bugzilla.xamarin.com/show_bug.cgi?id=59145
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	delegate IntPtr AVAudioIONodeInputBlock (uint frameCount);
```

Comment:
Please comment out this method until bug #59145 is fixed (or make it use AudioBufferList and then use `#if !WATCH` to exclude it from the watchOS build).

We don't want to expose bad API, since we can't fix it later.

The same goes for `AVAudioEngineManualRenderingBlock` further above.